wandb/sdk/internal/internal_api.py

```diff
@@ -3190,6 +3199,67 @@ def create_artifact_type(
         _id: Optional[str] = response["createArtifactType"]["artifactType"]["id"]
         return _id
 
+    def server_create_artifact_introspection(self) -> List:
```

Comment:
I'm still slightly leaning towards keeping the name for `server_create_artifact_introspection` since it's used to query the inputs for `createArtifact`. I'm using it to also query what inputs `useArtifact` but thats more out of not wanting to have to write another probe function. 

If there is a better way to query the graphql type Artifact that would be ideal.

Reply:
added lru cache